src/styles/utils/_normalize.scss

```diff
@@ -0,0 +1,17 @@
+body {
+  margin: 0;
+}
+* {
+  box-sizing: border-box;
+  font-family: "Roboto", Arial, sans-serif;
```

Comment:
font-family should be defined in body as it is inherited property

Reply:
I ailing elements by center? Do not see design or requirements in the task - how to align them (left, center, right, ...) ? Thanks